src/tests/fuzz/fuzz_applayerparserparse.c

```diff
@@ -0,0 +1,98 @@
+/**
+ * @file
+ * @author Philippe Antoine <contact@catenacyber.fr>
+ * fuzz harness for AppLayerProtoDetectGetProto
+ */
+
+
+#include "suricata-common.h"
+#include "app-layer-detect-proto.h"
+#include "flow-util.h"
+#include "app-layer-parser.h"
+
+#define HEADER_LEN 6
+
+void fuzz_openFile(const char * name) {
```

Comment:
what is this for?

Reply:
That is for onefile.c, a simple driver for which you can specify a output file
Let's wait a bit to see if we need it